app/components/Form/legoForm.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import { reduxForm, SubmissionError } from 'redux-form';
+import { handleSubmissionError } from './utils';
+import { pick, difference } from 'lodash';
+type Props = {
+  onSubmitFail: any => any,
+  onSubmit: any => Promise<*>,
+  /* Enable auto submissionError */
+  enableSubmissionError?: boolean,
+  /* Move the screen to the first error in the list on SubmissionError */
+  enableFocusOnError?: boolean,
+  /* Also pick these values based on the keys given on the array
+   * Should be used when there is no visible filed for the value,
+   * but that it should still pass the value further.
+   * 
+   * The id is always picked
+   */
+  pickAdditionalValues?: Array<string>,
```

Comment:
add `enablePicking`?? With a default of `true` or `false`?

Reply:
`enableValuePicking` is better imo. Default to false maybe? Better to explicitly handle it don't u think? Might be other arguments sent in addition to the registered ones, which could lead to issues if one is not aware of this picking.